x-pack/legacy/plugins/canvas/canvas_plugin_src/strings/functions/savedMap.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { savedMap } from '../../functions/common/savedMap';
+import { FunctionHelp } from '.';
+import { FunctionFactory } from '../../../types';
+
+export const help: FunctionHelp<FunctionFactory<typeof savedMap>> = {
+  help: i18n.translate('xpack.canvas.functions.savedMapHelpText', {
```

Comment:
Oh.  You *did* add the help documentation... you just didn't use it in the functions...?